src/api/typings/neonDB.d.ts

```diff
@@ -0,0 +1,75 @@
+import { Balance, Claims } from '../../wallet'
+import { RPCResponse } from '../../rpc'
+import { Fixed8 } from '../../utils'
+import { signingFunction, net, AssetAmounts } from './core';
+
+/** API Switch for MainNet and TestNet */
+export function getAPIEndpoint(net: net): string
+
+/** Get balances of NEO and GAS for an address */
+export function getBalance(net: net, address: string): Promise<Balance>
+
+/** Get amounts of available (spent) and unavailable claims. */
+export function getClaims(net: net, address: string): Promise<Claims>
+
+/** Gets the maximum amount of gas claimable after spending all NEO. */
+export function getMaxClaimAmount(net: net, address: string): Promise<Fixed8>
+
+/** Returns the best performing (highest block + fastest) node RPC. */
+export function getRPCEndpoint(net: net): Promise<string>
+
+/** Get transaction history for an account */
+export function getTransactionHistory(net: net, address: string): Promise<History>
+
+/** Get the current height of the light wallet DB */
+export function getWalletDBHeight(net: net): Promise<number>
+
+/** DEPRECATED: use claimGas instead */
+export function doClaimAllGas(
+  net: net,
+  privateKey: string
+): Promise<RPCResponse>
+
+/** DEPRECATED: use claimGas instead */
+export function doClaimAllGas(
+  net: net,
+  publicKey: string,
+  signingFunction: signingFunction
+): Promise<RPCResponse>
+
+/** Call mintTokens for RPX */
```

Comment:
All these `doX` functions are deprecated

Reply:
added `DEPRECATED` for `doX` methods